src/core/geometry/qgswkbtypes.h

```diff
@@ -142,6 +143,9 @@ class CORE_EXPORT QgsWkbTypes
       UnknownGeometry,
       NullGeometry
     };
+    Q_ENUM( GeometryType )
+    Q_DECLARE_FLAGS( GeometryTypes, GeometryType )
```

Comment:
Does this work? Wouldn't the enum values need to be bit flags (1,2,4,etc) for this to be useful? (And I don't think we can change that without breaking api...)

Reply:
Well... There is the solution of just changing the values here and hoping no-one is using their integer representations anywhere... ;) But I'd proceed with caution. There's always the chance that the int values have been used to save the state of some object in XML/qsettings.

Reply:
... or in plugins. Python is unfortunately very eager to print numbers instead of names for enum. What's the benefit we get from this change?

Reply:
the use of the proper Qt API instead of a list.
but that's the end of the story I guess, this will have to wait for 4.0.